packages/core/src/modules/action-menu/services/ActionMenuService.ts

```diff
@@ -9,6 +9,8 @@ import type {
   CreateRequestOptions,
   FindMenuOptions,
 } from './ActionMenuServiceOptions'
+// eslint-disable-next-line no-restricted-imports
+import type { Query } from 'packages/core/src/storage/StorageService'
```

Comment:
To avoid these eslint errors you should use relative paths from each file. Something like:

```suggestion
// eslint-disable-next-line no-restricted-imports
import type { Query } from '../../../storage/StorageService'
```